src/js/samples/components/Item/Item.tsx

```diff
@@ -0,0 +1,201 @@
+    workflows: SampleWorkflows;
+    onSelect: (id: string, index: number, shiftKey: boolean) => void;
+    onQuickAnalyze: (id: string) => void;
+};
+
+/**
+ * A condensed sample item for use in a list of samples
+ *
+ * @param id - The unique identifier of the sample
+ * @param index - Index of the sample
+ * @param checked - Whether the sample is selected
+ * @param handle - The user who created the sample
+ * @param created_at - The date the sample was created
+ * @param name - Name of the sample
+ * @param library_type - Library type associated with the sample
+ * @param ready - Whether the sample is ready
+ * @param workflows - Workflows associated with the sample
+ * @param labels - Labels associated with the sample
+ * @param job - Information about the job associated with the sample
+ * @param onSelect - Callback to handle sample selection
+ * @param onQuickAnalyze - Callback to trigger quick analysis
+ */
+export function SampleItem({
+    id,
+    index,
+    checked,
+    handle,
+    created_at,
+    name,
+    library_type,
+    ready,
+    workflows,
+    labels,
+    onSelect,
+    onQuickAnalyze,
+}: SampleItemProps) {
+    const handleCheck = e => {
+        onSelect(id, index, e.shiftKey);
+    };
+
+    const handleQuickAnalyze = () => {
+        onQuickAnalyze(id);
+    };
+
+    return (
+        <StyledSampleItem>
+            <SampleItemCheckboxContainer onClick={handleCheck}>
+                <Checkbox checked={checked} />
+            </SampleItemCheckboxContainer>
+
+            <SampleItemData>
+                <SampleItemMain>
+                    <SampleItemTitle>
+                        <Link to={`/samples/${id}`}>{name}</Link>
+                        <Attribution time={created_at} user={handle} />
+                    </SampleItemTitle>
+                </SampleItemMain>
+                <SampleItemLabels>
+                    <SampleLibraryTypeLabel libraryType={library_type} />
+                    {labels.map(label => (
+                        <SmallSampleLabel key={label.id} {...label} />
+                    ))}
+                </SampleItemLabels>
+            </SampleItemData>
+            <SampleItemWorkflows>
+                <WorkflowTags id={id} workflows={workflows} />
+            </SampleItemWorkflows>
+            <SampleItemIcon>
+                <EndIcon ready={ready} onClick={handleQuickAnalyze} />
+            </SampleItemIcon>
+        </StyledSampleItem>
+    );
+}
+
+export function mapStateToProps(state, ownProps) {
+    const { id, ready, index, user, created_at, name, library_type, workflows, labels } = find(
+        state.samples.documents,
+        {
+            id: ownProps.id,
+        },
+    );
+
+    const checked = getIsSelected(state, ownProps.id);
+
+    return {
+        id,
+        ready,
+        index,
+        handle: user.handle,
+        created_at,
+        name,
+        library_type,
+        workflows,
+        labels,
+        checked,
+    };
```

Comment:
Extra variable not necessary.
```suggestion
    return {
        id,
        ready,
        index,
        handle: user.handle,
        created_at,
        name,
        library_type,
        workflows,
        labels,
        checked: getIsSelected(state, ownProps.id),
    };
```